src/mesh/FloodingRouter.cpp

```diff
@@ -45,7 +51,7 @@ void FloodingRouter::sniffReceived(const meshtastic_MeshPacket *p, const meshtas
     }
     if (!isToUs(p) && (p->hop_limit > 0) && !isFromUs(p)) {
         if (p->id != 0) {
-            if (config.device.role != meshtastic_Config_DeviceConfig_Role_CLIENT_MUTE) {
+            if (isRebroadcaster()) {
```

Comment:
Maybe we should change the debug log on line 65 as well.

Reply:
Good call. Pushed